### ISSUE TITLE
[Accessibility]A11y_ Java Tooling for IntelliJ/Eclipse_ Eclipse Azure Device login_Keyboard: Keyboard trap is observed for while navigating from the error details edit popup

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/azure-tools-for-java#3322]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS2.1.2

**Repro Steps:** 
1) Open Java EE - Eclipse application
2) Install 'Azure Toolkit for Eclipse' from Marketplace
3) Tab till tools-->Azure-->Azure Sign in and press enter
4) Select 'Device login' and tab till sign in and press enter
5) Tab till cancel button and press enter
6) Verify if there is any keyboard trap while navigating form the error details edit field 

**Actual Result:** 
Keyboard trap is observed for while navigating from the error details edit popup

**Expected Result:** 
There should not be any keyboard trap in the popup. Focus should go from the error details to the ‘OK’ button

**User Impact:** 
Keyboard users will find difficulty in coming out of the popup if there is any keyboard trap in the error details section

**MAS Reference:** 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={ebdc3fe1-09a9-4aa2-a29b-e97764774e6c}

[Bug_7_keyboard trap issue.pptx](https://github.com/microsoft/azure-tools-for-java/files/3451182/Bug_7_keyboard.trap.issue.pptx)

**Comment by Flanker32** [comment]
Fixed in #3494

**Comment by Flanker32** [comment]
@Tanushagottipati Please have a review.

**Comment by Tanushagottipati** [comment]
Regressed in the below build and the issue is fixed. Hence closing the bug and attached video for reference
Environment Details:
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17
[Uploading Bug fixed2.pptx…]()

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749799
